### ISSUE TITLE
Create using a managed image ?

### ISSUE THREAD

**Issue by andyt530** [Azure/azure-cli#7531]
Could do with an example of how to create a SIG image definition from an existing managed image.  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c1ea8a20-1ce0-a34c-872e-8101f1494479
* Version Independent ID: ed92d5cf-a13b-de7c-9693-337e7c20f3ae
* Content: [az sig image-definition](https://docs.microsoft.com/en-us/cli/azure/sig/image-definition?view=azure-cli-latest#az-sig-image-definition-create)
* Content Source: [latest/docs-ref-autogen/sig/image-definition.yml](https://github.com/Azure/azure-docs-cli-python/blob/live/latest/docs-ref-autogen/sig/image-definition.yml)
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by adewaleo** [comment]
TL;DR
Shared Image Gallery (sig) **Image Definitions** are not created from managed images, **Image Versions** on the other hand are.  

Examples on how to create image-versions are shown [here](https://docs.microsoft.com/en-us/cli/azure/sig/image-version?view=azure-cli-latest#az-sig-image-version-create).

Here is one based one of the (modified) examples. Create an image version from an image definition and managed image. `--managed-image` can be a name or id:
```
az sig image-version create -g MyResourceGroup --gallery-name MyGallery \
--gallery-image-definition MyImage --gallery-image-version 1.0.0 \
--managed-image MyManagedImage --no-wait
```

More info:

Image definitions can be thought of as a definition of a class / grouping of **image versions** which are created from managed images. VM's cannot be created from **image definitions** but can be created from **image versions**.

According to the Shared Image Gallery (sig) doc [overview](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/shared-image-galleries):
- A managed image is a "baseline image that can be used alone or used to create multiple shared image versions in an image gallery."
- A gallery image (definition) includes information such as: "whether the image is Windows or Linux, release notes, and minimum and maximum memory requirements. This type of image is a resource within the Resource Manager deployment model, but it isn't used directly for creating VMs. It is a definition of a **type** of image."
- An image version, however:  "is what you use to create a VM when using a gallery. You can have multiple versions of an image as needed for your environment. Like a managed image, when you use an image version to create a VM, the image version is used to create new disks for the VM. Image versions can be used multiple times."

**Comment by adewaleo** [comment]
I hope this helps.
